src/app/+dashboard/components/exoft-achievements/exoft-achievements.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="ach">
+  <div class="h-ach">
+    <p>EXOFT ACHIEVEMENTS</p>
```

Comment:
It is better if you write this text in lowercase and provide appropriate class to display it in uppercase

So for example in styles.scss class you will have
```
.text-uppercase {
  text-transform: uppercase;
}
```
And here you will have 
`<p class="text-uppercase">exoft achievements</p>`

Its up to you to change it or not

Reply:
fixed